### PR TITLE
Fix tutorial 1 test runner

### PR DESCRIPTION
**Pull request by mitschabaude**
Fixes tutorial 1 CI test by adding back the zkapp cli installation command which the tutorial runner needs to execute to get things started.

Also, I removed the disclaimer "this tutorial was last tested with versions ...", since the tutorial runner CI for tutorial 1 is actually always tested with the latest versions in CI

### CODE REVIEW

**Review by ymekuria — APPROVED** [review]
Thanks for fixing CI @mitschabaude .

File: docs/zkapps/tutorials/01-hello-world.mdx [hunk 1]
```diff
@@ -48,10 +48,14 @@ To prevent copying line numbers and command prompts as shown in the examples, us
 
 ## Prerequisites
 
-This tutorial has been tested with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.7.5` and [SnarkyJS](https://www.npmjs.com/package/snarkyjs) `0.9.8`.
-
 Ensure your environment meets the [Prerequisites](/zkapps/tutorials#prerequisites) for zkApp Developer Tutorials.
 
+In particular, make sure you have the zkapp CLI installed:
```
Comment (barriebyron) [inline docs/zkapps/tutorials/01-hello-world.mdx:53]:
```suggestion
In particular, make sure you have the zkApp CLI installed:
```

**Review by barriebyron — APPROVED** [review]
thank you @mitschabaude for this update. I added the same update to Tutorial 2

**Review by barriebyron — APPROVED** [review]
thank you @mitschabaude  for this fix! I added the same update to tutorial 2

**Pull request merged**